test/app/webpack.config.js

```diff
@@ -94,7 +108,6 @@ const webpackConfig = {
 // With env var set, run against built ESM module in dist folder
 if (DIST_ESM) {
   webpackConfig.resolve.alias['./getOktaSignIn'] = './getOktaSignInFromNPM';
-  webpackConfig.resolve.alias['@okta/okta-signin-widget'] = path.resolve(WORKSPACE_ROOT, 'dist/');
```

Comment:
Instead of this alias, add 
`"@okta/okta-signin-widget": "link:../../dist"`
to dependencies in [test app's `package.json`](https://github.com/okta/okta-signin-widget/pull/2931/files#diff-3ead3416e777ea58d2d2fae632649a85afbe063f9cbd29e952d17d205733288eR12).

Thus [import css](https://github.com/okta/okta-signin-widget/pull/2931/files#diff-29a7bf42fe2a323c33d18ce9ef81939578d93a19a5e5462364f1ebf0f3cb8e34R2) is resolving properly using `package.json` in `dist`.

But this requires adding [`resolve.conditionNames = ['browser', 'import', 'default']`](https://github.com/okta/okta-signin-widget/pull/2931/files#diff-9f6df93a8ee584559d26d6acfc3d7b3da5fd08e2cf8134429c09676a761597f8R38) in order to resolve ESM version of SIW bundle from dist.